opa-audit-operator/src/manifests.py

```diff
@@ -0,0 +1,124 @@
+import logging
```

Comment:
I tried this, but it didn't work. 
Since the parent directory is not in the same module I couldn't simply import it, to overcome this I had to create a manifests dir like this:
```console
charms/
├── manifests_base
├── opa-audit-operator
└── opa-manager-operator
```
Then I tried to:
1) Add the parent directory to `os.path`, but charmcraft pack wasn't packing the parent dir so it failed.
2) Add a symlink from the parent/ dir to the src dir and import from the symlink, but charmcraft still didn't pack it.

Creating a hard link instead of a symlink works, but I don't think we can push hard links to git and also we would have to create a link for each file (you can't hard link a dir). I couldn't think of any other way to make it work, it seems like we will have to keep it like this

Reply:
according the the charmcraft folks, you can hack it in with:
```
parts:
  charm:
    prime: 
      - extra-magic.txt
    override-pull: |
      cp /tmp/extra-magic.txt $CRAFT_PART_SRC/
      craftctl default
```
but you must use destructive mode. Since destructive mode is required, its probably better to just have the duplicated files than be forced to build locally like that.